pom.xml

```diff
@@ -1814,6 +1814,29 @@ flexible messaging model and an intuitive client API.</description>
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
```

Comment:
Instead of adding a separate profile for JDK17, couldn't we update the existing JDK11 profile to cover both? /Cc @eolivelli @nicoloboschi

Reply:
In my opinion, it doesn’t add much complexity, but it’s clearer than putting it together. (what's jdk 11 need, what's jdk17 need)

Reply:
How is it different for JDK17? It's better to combine if we can. There will be new JDK versions coming frequently and we don't want to add a new profile for each one of them.

Reply:
Currently we can combine this to a line, should I combine this ? We can only add profile for jdk LTS and latest version. cc @codelipenghui

Reply:
+1 for not adding new profile, to me it's useful only if something not related only to the tests is different